script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt

```diff
@@ -837,22 +894,26 @@ Senpai speaks to me from behind.
 [sha:07835edc547a137051f64dfb2d622e9f96d4cd27]{
 -- Page 301, Offset 34065.
 -- 「まだ解らないんですか。貴方は今どのくらいの速さで走ったと思っているんです。致命傷を受けて、どうして死なないでいるんです。
-"You still don't get it, do you? Do you not know how fast you were running just now? Do you not know why you haven't died even after sustaining injuries that should've been fatal?
+// Sir you were doing 70 in a school zone
+"Don't you get it yet?
+ How fast do you think you were running just then?
+ How do you think you're still alive after sustaining all those injuries?
 }
 [sha:b43aebe5775a4c80dd7ad461134a66d4980dc23c]{
 -- Page 301, Offset 34066.
 -- ―――貴方は箱の中のマウスと変わらない。
-―――You're no different from a mouse trapped in a box.
+// Despite all his rage
+You're nothing but a cornered rat.
 }
 [sha:58eb57a107e5e2528d75b5c493c94ba5383cdac1]{
 -- Page 301, Offset 34067.
 -- なのに、まだ、何から逃げ出そうというんですか？」
-But despite that, you still intend on continuing to try to run away?"
+And besides, just what is it that you're trying to run from?"
```

Comment:
I think her argument is something like "you're already a vampire, so being in this city, you're trapped already, why are you running?"

>You're practically a mouse already caught in its trap.

>So then what is it exactly you're trying to run away from?